src/containers/About/About.test.js

```diff
@@ -39,39 +40,40 @@ describe('About', () => {
     expect(queryByText('PipelineVersion')).toBeTruthy();
     expect(queryByText('v0.10.0')).toBeTruthy();
     expect(queryByText('IsOpenShift')).toBeTruthy();
+    expect(queryByText('ReadOnly')).toBeTruthy();
     expect(queryByText('True')).toBeTruthy();
   });
 
   it('should render error when an expected property is missing', async () => {
     const installProperties = Promise.resolve({
       InstallNamespace: 'tekton-pipelines',
-      DashboardVersion: '',
+      // DashboardVersion: '', this is intentionally missing
       PipelineVersion: 'v0.10.0',
-      IsOpenShift: false
+      IsOpenShift: 'false',
```

Comment:
shouldn't these be booleans to match the other tests and the updated response from the back end?

Reply:
Yep, sorted